code/modules/mob/living/living_emote.dm

```diff
@@ -100,6 +100,11 @@
 
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
+		// For masks that give unique death sounds
+		if(istype(H) && isclothing(H.wear_mask) && H.wear_mask.unique_death)
+			playsound(H, H.wear_mask.unique_death, 200, TRUE, TRUE)
```

Comment:
This should really just return `H.wear_mask.unique_death` instead of playing it in `get_sound()`. This defaults to a 17 tile range instead of following other deathgasps.